benchmark/reformat_input_gi.py

```diff
@@ -0,0 +1,143 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+"""
+Reformat input genome files to formats required by given
+GI tool
+=========================================================
+"""
+
+import click
+from os import remove
+from os.path import join
+
+from skbio import Sequence, DNA
+
+
+def reformat_egid(genbank_fp,
+                  output_dir):
+    """ Reformat input genome to the formats accepted by EGID.
+
+    Parameters
+    ----------
+    genbank_fp: string
+        file path to genome in GenBank format
+    output_dir: string
+        output directory path
+
+    Notes
+    -----
+    Input to EGID are five obsolete NCBI standard files: gbk, fna, faa, ffn
+    and ptt.
+    """
+    loci = []
+    nucl_seq = ''
+    genes = {}
+    nseq = 0  # number of nucleotide sequences
+    with open(genbank_fp, 'r') as input_f:
+        for line in input_f:
+            if line.startswith('//'):
+                nseq += 1
+    abs_pos = 0  # absolute position in concantenated nucleotide sequence
+    for i in range(nseq):
+        gb = Sequence.read(genbank_fp, seq_num=i+1, format='genbank')
```

Comment:
Perhaps I'm missing something but will it be easier to remove lines 38->48 and simply do:
```
for gb in Sequence.read(genbank_fp, format='genbank'):
```
? I think it will be much faster too.